src/snyk/common/configuration/configuration.ts

```diff
@@ -167,7 +167,12 @@ export class Configuration implements IConfiguration {
   }
 
   async getToken(): Promise<string | undefined> {
-    return SecretStorageAdapter.instance.get(SNYK_TOKEN_KEY);
+    try {
+      return SecretStorageAdapter.instance.get(SNYK_TOKEN_KEY);
+    } catch (e) {
+      // if the token cannot be parsed then clear the token
```

Comment:
Should we additionally log the error to output channel? It's still a mystery when this fails by default.

Reply:
Yeah I thought about adding a logger here but then the only error we will be catching is the [`Cannot get password` ](https://github.com/microsoft/vscode/blob/main/src/vs/workbench/api/browser/mainThreadSecretState.ts#L48) which isn't very useful to start with.